recipes/hdf5storage/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "hdf5storage" %}
+{% set version = "0.1.14" %}
+{% set sha256 = "180593f91325c952346010dde32b359563f69d2760b6e7d25a8d2d851b93b1ab" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.zip
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.zip
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+    - h5py >=2.3
```

Comment:
Technically only [`h5py >=2.1`]( https://github.com/frejanordsiek/hdf5storage/blob/0.1.14/setup.py#L22 ) is required, but `h5py >=2.3` provides [full functionality]( https://github.com/frejanordsiek/hdf5storage#dependencies ). Given that `h5py` 2.3 is already 3yrs old (2.1 is 5yrs old) and `conda-forge` started with 2.6, this not only totally reasonable, but maybe even unnecessary to specify. Though we put here anyways as a note.